apps/sentry-client-desktop/src/features/keys/HasKeys.tsx

```diff
@@ -27,12 +27,15 @@ interface HasKeysProps {
 	combinedLicensesMap: LicenseMap,
 	statusMap: StatusMap,
 	isWalletAssignedMap: WalletAssignedMap,
+	operatorWalletData: BulkOwnerOrPool[]
 }
 
-export function HasKeys({combinedOwners, combinedLicensesMap, statusMap, isWalletAssignedMap}: HasKeysProps) {
+export function HasKeys({combinedOwners, combinedLicensesMap, statusMap, isWalletAssignedMap, operatorWalletData}: HasKeysProps) {
 	const setDrawerState = useSetAtom(drawerStateAtom);
 	const setModalState = useSetAtom(modalStateAtom);
 	const {data, setData} = useStorage();
+
+	//TODO this should be attached to operatorWalletData from the subgraph when the accrued / claimed fields have been added to the graph with #188210182
```

Comment:
Is this TODO still needed? If so lets make a chore ticket for it

Reply:
I added the ticket id at the end of the comment, this TODO can be addressed once the subgraph story is completed.